script/script.js

```diff
@@ -0,0 +1,35 @@
+setInterval(function() {
+    var audio = document.getElementById("myAudio");
+    audio.currentTime = 0;
+    audio.play();
+}, 3000);
+
+setInterval(function() {
+    var audio = document.getElementById("myAudio");
```

Comment:
in modern js we are trying to avoid using the `var`, you can use `let` or `const` instead